core/txpool/blobpool/blobpool.go

```diff
@@ -87,8 +87,9 @@ type blobTxMeta struct {
 	hash    common.Hash   // Transaction hash to maintain the lookup table
 	vhashes []common.Hash // Blob versioned hashes to maintain the lookup table
 
-	id   uint64 // Storage ID in the pool's persistent store
-	size uint32 // Byte size in the pool's persistent store
+	id          uint64 // Storage ID in the pool's persistent store
+	size        uint32 // Byte size in the pool's persistent store
+	encodedSize uint64 // RLP-encoded size of transaction
```

Comment:
I would rename encodedSize to size and size to persistentSize or something

Reply:
I've changed encodedSize to size and size to storageSize